features/attached_status.feature

```diff
@@ -19,6 +19,39 @@ Feature: Attached status
            | jammy   |
            | kinetic |
 
+    @series.xenial
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: Non-root status can see in-progress operations
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I attach `contract_token` with sudo
+        When I run shell command `sudo pro enable cis >/dev/null & pro status` as non-root
+        Then stdout matches regexp:
+        """
+        NOTICES
+        Operation in progress: pro enable
+        """
+        When I run `pro status --wait` as non-root
+        When I run `pro disable cis --assume-yes` with sudo
+        When I run shell command `sudo pro enable cis & pro status --wait` as non-root
```

Comment:
Nice and confims that we block an emit "...." as non-root.
Also we could verify just before this something like:
`sudo pro enable esm-apps --assume-yes  & pro status --format json | jq -r .execution_status`
matches "active" instead of "inactive"

Reply:
Good idea! I'll add it

Reply:
Added!